test/integration/pipeline/pipeline_test.cpp

```diff
@@ -193,3 +200,33 @@ TEST(PipelineIntegrationTest, SendTxSequenceAwait) {
       .sendTxSequenceAwait(tx_sequence, check_block)
       .done();
 }
+
+/**
+ * Check that after no transactions were committed we are able to send and apply
+ * new transactions
+ *
+ * @given createFirstDomain and createSecondDomain transactions
+ * @when first domain is created second time
+ * @then block with no transactions is created @and after that
```

Comment:
Other tests use slightly different style, with "AND" on a new line. Maybe use it here too?